pkg/api/junit.go

```diff
@@ -0,0 +1,121 @@
+package api
```

Comment:
Crating the `api` package as this lib can be used by plugins too.
Should we need to place under `api/junit` directory creating a dedicated package?